ush/forecast_postdet.sh

```diff
@@ -0,0 +1,959 @@
+  fi
+  if [ $ICERES = '050' ]; then
+    ICERESdec="0.50"
+  fi 
+  if [ $ICERES = '100' ]; then
+    ICERESdec="1.00"
+  fi
+  
+  ice_grid_file=${ice_grid_file:-"grid_cice_NEMS_mx${ICERES}.nc"}
+  ice_kmt_file=${ice_kmt_file:-"kmtu_cice_NEMS_mx${ICERES}.nc"}
+  
+  iceic="cice_model.res_$CDATE.nc"
+  
+  # Copy CICE IC 
+  $NCP -p $ICSDIR/$CDATE/ice/cice_model_${ICERESdec}.res_$CDATE.nc $DATA/$iceic
+  
+  echo "Link CICE fixed files"
+  $NLN -sf $FIXcice/$ICERES/${ice_grid_file} $DATA/
+  $NLN -sf $FIXcice/$ICERES/${ice_kmt_file} $DATA/
+  $NLN -sf $FIXcice/$ICERES/$MESH_OCN_ICE $DATA/
+}
+
+CICE_nml() {
+  echo "SUB ${FUNCNAME[0]}: Creating name list for CICE"
+  source $SCRIPTDIR/parsing_namelists_CICE.sh
+  CICE_namelists
+}
+
+CICE_out() {
+  echo "SUB ${FUNCNAME[0]}: Copying output data for CICE"
+  if [ $inistep = 'cold' ]; then
+    echo "mediator cold start, no copying of data for CICE"
+  else
+    export ENSMEM=${ENSMEM:-01}
+    export IDATE=$CDATE
+    $NCP -p $DATA/ice_in $COMOUT/
+    if [ $FHRGRP -eq 0 ]; then
+      fhrlst="anl"
+    else
+      fhrlst=$(echo $FHRLST | sed -e 's/_/ /g; s/\[/ /g; s/\]/ /g; s/f/ /g; s/,/ /g')
+    fi
+  
+    for fhr in $fhrlst; do
+      export fhr=$fhr
+      #  --------------------------------------
+      #  cp cice data to COMOUT directory
+      #  --------------------------------------
+      YYYY0=`echo $IDATE | cut -c1-4`
+      MM0=`echo $IDATE | cut -c5-6`
+      DD0=`echo $IDATE | cut -c7-8`
+      HH0=`echo $IDATE | cut -c9-10`
+      SS0=$((10#$HH0*3600))
+
+      VDATE=$($NDATE $fhr $IDATE)
+      YYYY=`echo $VDATE | cut -c1-4`
+      MM=`echo $VDATE | cut -c5-6`
+      DD=`echo $VDATE | cut -c7-8`
+      HH=`echo $VDATE | cut -c9-10`
+      SS=$((10#$HH*3600))
+
+      if [[ 10#$fhr -eq 0 ]]; then
+        $NCP -p $DATA/history/iceh_ic.${YYYY0}-${MM0}-${DD0}-`printf "%5.5d" ${SS0}`.nc $COMOUT/iceic$VDATE.$ENSMEM.$IDATE.nc
+        status=$?
+        [[ $status -ne 0 ]] && exit $status
+        echo "fhr is 0, only copying ice initial conditions... exiting"
+      else
+        $NCP -p $DATA/history/iceh_`printf "%0.2d" $FHOUT`h.${YYYY}-${MM}-${DD}-`printf "%5.5d" ${SS}`.nc $COMOUT/ice$VDATE.$ENSMEM.$IDATE.nc
+        status=$?
+        [[ $status -ne 0 ]] && exit $status
+      fi
+    done                        
+  fi
+}
+
+GOCART_rc() {
+  echo "SUB ${FUNCNAME[0]}: Linking input data and copying config files for GOCART"
+  # set input directory containing GOCART input data and configuration files
+  # this variable is platform-dependent and should be set via a YAML file
+  
+  # link directory containing GOCART input dataset, if provided
+  if [ ! -z "${CHM_INPDIR}" ]; then
+    $NLN -sf ${CHM_INPDIR} $DATA
+    status=$?
+    [[ $status -ne 0 ]] && exit $status
+  fi
+  
+  # copying GOCART configuration files
+  if [ ! -z "${CHM_CFGDIR}" ]; then
+    $NCP     ${CHM_CFGDIR}/*.rc $DATA
+    status=$?
+    [[ $status -ne 0 ]] && exit $status
+  fi
+}
+
+GSD_in() {
```

Comment:
GSD_in GSD_nml and GSD_out were all removed in f/c-c and should be removed here too.

Reply:
resolved